pkg/odo/cli/registry/util/util.go

```diff
@@ -34,3 +35,11 @@ func IsSecure(registryName string) bool {
 
 	return isSecure
 }
+
+func IsGitBasedRegistry(url string) bool {
+	return strings.Contains(url, "github")
```

Comment:
The searched string could be extended (to `github.com/`, or `https://github.com/`, ...) to avoid some false positive when people have `github` in the path of their URL

Reply:
makes sense, updating